packages/edit-site/src/components/page-patterns/grid-item.js

```diff
@@ -122,7 +122,12 @@ export default function GridItem( { categoryId, composite, icon, item } ) {
 					aria-label={ item.title }
 					aria-describedby={
 						ariaDescriptions.length
-							? ariaDescriptions.join( ' ' )
+							? ariaDescriptions
+									.map(
+										( _, index ) =>
+											`${ descriptionId }-${ index }`
```

Comment:
Minor: `${ descriptionId }-${ index }` is used one more time, a few lines below, and could be set to a variable.

Reply:
These two uses of `${ descriptionId }-${ index }` are in separate loops so I'm not sure they can be extracted to a shared variable.

Reply:
Right. Aside: I'm not even sure why there is the need to push the notifications/IDs to an array. Is there any case where a pattern can have more than one description?